Source/WebCore/loader/cache/CachedResourceRequest.cpp

```diff
@@ -128,66 +128,44 @@ void CachedResourceRequest::setDomainForCachePartition(const String& domain)
     m_resourceRequest.setDomainForCachePartition(domain);
 }
 
-static constexpr ASCIILiteral acceptHeaderValueForWebPImageResource()
+static inline void appendAcceptHeaderValueForAdditionalSupportedImageMIMETypes(StringBuilder& builder)
```

Comment:
This is a really long name but I think it is okay. Maybe something like this can be used:

```
static inline void appendAdditionalSupportedImageMIMETypes(StringBuilder& acceptHeader)
```